src/pages/careers/internships.js

```diff
@@ -9,19 +9,38 @@ import Footer from "../../sections/General/Footer";
 import InternshipPage from "../../sections/Careers/Careers-Internship-grid";
 
 import { GlobalStyle } from "../../sections/app.style";
-import theme from "../../theme/app/themeStyles";
+import darktheme from "../../theme/app/darkThemeStyles";
+import lighttheme from "../../theme/app/themeStyles";
+import { useCookies } from "react-cookie";
+import { useEffect } from "react";
+import { useState } from "react";
```

Comment:
Both of these could be imported under the same line as `import React from "react"`. 
Update this in other places too.

Reply:
Yes, I will refactor this code once It fills the purpose, I made this PR just to show approach :)